state/ksm/init.sls

```diff
@@ -0,0 +1,12 @@
+# Install and start sysfsutils
+#
+sysfsutils:
+  pkg.installed:
+    - name: sysfsutils
+
+  service.running:
+    - name: sysfsutils
+    - enable: True
+    - watch:
+      - file: /etc/sysfs.conf
+      - file: /etc/sysctl.d/*
```

Comment:
You can use the `sysctl.present` state which sets and persists sysctl variables.